linq4j/src/main/java/org/apache/calcite/linq4j/tree/ConstantExpression.java

```diff
@@ -245,6 +249,31 @@ private static ExpressionWriter map(ExpressionWriter writer, Map map,
     return writer.append(end);
   }
 
+  private static ExpressionWriter writeSet(ExpressionWriter writer, Set set) {
+    writer.append("com.google.common.collect.ImmutableSet.");
+    if (set.isEmpty()) {
```

Comment:
Instead of hard code the package name, can we use `XXX.class.getCanonicalName()` ?